src/utils.ts

```diff
@@ -663,6 +664,9 @@ export function uuidV4(): Buffer {
  */
 export function maxWireVersion(topologyOrServer?: Connection | Topology | Server): number {
   if (topologyOrServer) {
+    if (topologyOrServer.loadBalanced) {
+      return MAX_SUPPORTED_WIRE_VERSION;
```

Comment:
We are assuming that load balancer is running against the latest server for the time being. If the server starts backporting support, which it might, then this will need to change.

Reply:
Would there be an issue when the server is updated to a later version? I'm not 100% clear on the implications of setting maxWireVersion higher than the "actual" max wire version of the underlying server

Reply:
Based on our conversation and the fact that this is something that it sounds like was an acceptable risk per the spec, can we add a comment here briefly mentioning the potential issues and that this is intentional?

Reply:
Added comment to the check.